LDM-148.tex

```diff
@@ -158,6 +158,31 @@ \section{Summary Concept of Operations}\label{summary-concept-of-operations}
 computing environments including the Data Access Centers, acting as the spine that
 supports them all.
 
+\subsection{Data Flow}\label{data-flow}
+
+Images (science and calibration) are taken on the mountain and stored in the camera image buffer (sized for a week or more).
+Images are copied by the camera software to the Camera Diagnostic Cluster for rapid analysis.
+Images may reside on that cluster for a short time.
+Images are also transferred to the Observatory Operations Data Service (OODS) for use by the OCS-Controlled Pipeline Service (OCPS), the Commissioning Cluster, and other Observatory systems such as the Active Optics System.
+They are expected to reside in the OODS for at least 30 days.
+At the time of readout, images are sent quickly to the US Data Facility (USDF) for Prompt Processing.
+A copy of the raw images is kept in the Base Facility in Chile as part of the Data Backbone, and an archival copy is also maintained in the Data Backbone at the US Data Facility.
+After an agreed delay period, the raw images are made available to the data rights holders in both the US and Chilean Data Access Centers (DACs) via the LSST Science Platform.
+During the 24 hours after readout, raw images are sent to the Satellite Data Facility at CC-IN2P3 both as a disaster recovery backup and to facilitate the Data Release Processing there at a later date.
+
+Information about the state of the Observatory (parts of which are sometimes referred to as image metadata) is collected from messages in the Observatory control network and transmitted via two mechanisms.
+The Header Service component of the Archiving service collects information required for the OCPS and Prompt Processing and creates FITS headers that are attached to the image pixels during export from the camera image buffer.
+Separately, the Engineering and Facility Database (EFD) collects all control messages and transports them via Kafka messaging to a replica at the US Data Facility, where they are aggregated and summarized for eventual use by the Calibration Products Production.
+The EFD data is made available to the US and Chilean DACs as well.
+
+Alerts generated by the Alert Production payload in the Prompt Processing service are distributed as rapidly as possible by the Alert Distribution service.
```

Comment:
Suggest: "Alerts generated by the Alert Production payload in the Prompt Processing service are distributed as rapidly as possible by the Alert Distribution service. This is currently envisioned to be 60s after the image is taken in Chile."